test/modules/.eslintrc.js

```diff
@@ -0,0 +1,27 @@
+// For reference: https://github.com/airbnb/javascript
+
+module.exports = {
+  env: {
+    mocha: true,
+    browser: true,
+  },
+
+  "globals": {
+    check: true,
+    gen: true,
+  },
+
+  parserOptions: {
+    sourceType: 'module',
+  },
+
+  rules: {
+    // We still get the value of this rule, it just allows for dev deps
+    'import/no-extraneous-dependencies': ['error', {
+      devDependencies: true
+    }],
+
+    // We want to keep each test structured the same, even if its contents are tiny
+    'arrow-body-style': 'off',
```

Comment:
Does this mean no single-expression arrow functions in tests? Could we do `as-needed`?

Reply:
No, it means that eslint doesn't care about what kind of function style you use in our tests. I copied that from `test/.eslintrc.js` so you are absolutely free to change it.